src/commands/preserve-auth0-cookies.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Preserves cookies from the Auth0 session.
+ *
+ * ## Docs
+ * @see https://docs.cypress.io/api/cypress-api/cookies#Preserve-Once
+ *
+ * inspired by
+ * @see https://github.com/sir-dunxalot/cypress-nextjs-auth0/issues/23#issuecomment-958144472
+ */
+Cypress.Commands.add('preserveAuth0CookiesOnce', () => {
```

Comment:
To match the method name, should the filename be `preserve-auth0-cookies-once.js`?